third_party/VENDOR-LICENSE

```diff
@@ -1802,13 +1802,26 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 Import: github.com/tektoncd/pipeline/vendor/github.com/gobuffalo/envy
 
 The MIT License (MIT)
-Copyright (c) 2018 Mark Bates
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) 2019 Mark Bates
```

Comment:
Were these vendor changes intentional?

Reply:
I ran `./hack/update-codegen.sh` which updated this file

Reply:
Ah strange, looks innocuous anyway...